spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -64,7 +64,7 @@ An alternative is selected for the bean archive if either:
 ----
 <beans xmlns="http://xmlns.jcp.org/xml/ns/javaee"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-       xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd">
+       xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd" bean-discovery-mode="all" version="1.1">
```

Comment:
It would make sense for the spec to reference the current version of its assets, e.g. the 2.0 xsd and use version 2.0.